packages/styles/scss/components/feature-card-block-large/_feature-card-block-large.scss

```diff
@@ -134,6 +134,10 @@ $fcb-breakpoint-up--lg: map-get(
       @include carbon--type-style('productive-heading-05');
     }
 
+    .#{$prefix}--card__footer {
+      flex-direction: row-reverse;
+    }
```

Comment:
Thank you for your thoughts @IgnacioBecerra! In case I was not clear enough (my apologies if it's the case), here's what I'm actually suggesting. Please let me know if it doesn't work. Thanks!

```suggestion
```

Reply:
Hmm, it seems removing it has no effect, the arrow stays on the left unless `row-reverse` is used. It's also used in the other Feature Cards as well.

Reply:
Let's move forward with the existing solution.

Reply:
@IgnacioBecerra @jeffchew This code seems to have caused a visual regression in React. Let's see how we can resolve the arrow icon position in another way.

Reply:
it looks like it moved the arrow to the lower right instead of lower left, that's the intention right?
<img width="1636" alt="Screen Shot 2020-10-01 at 6 34 48 PM" src="https://user-images.githubusercontent.com/1641214/94870141-d091b880-0414-11eb-8c83-1c5a0cffdab3.png">

Reply:
Oh totally missed that the visual change is intentional. Please disreegard my earlier messages. My sincere apologies on this.